libbinlogevents/include/binlog_event.h

```diff
@@ -129,7 +129,8 @@
    1 + 255 /* host_len, host */ + 1U + 1 /* type, explicit_def..ts*/ + 1U +   \
    8 /* type, xid of DDL */ + 1U +                                            \
    2 /* type, default_collation_for_utf8mb4_number */ +                       \
-   1 /* sql_require_primary_key */ + 1 /* type, default_table_encryption */)
+   1 /* sql_require_primary_key */ + 1 /* type, default_table_encryption */ + \
```

Comment:
Is this part OK? starting from sql_require_primary_key we are storing type + value, so always 2 bytes.
Note that for variables before sql_require_primary_key there is 
`1U + xxx /* type, something */`
but starting from  sql_require_primary_key the type if not counted. Shouldnt it be like:
1 + 1 /* type, sql_require_primary_key */ \
+ 1 + 1 /* type, default_table_encryption */ + \
1 + 1 /* type, binlog_ddl_skip_rewrite */

Reply:
Fixed